### PR TITLE
Added compatibility matrix and API to fetch GQL server version

### PR DESCRIPTION
**Pull request by Saranya-jena**
Added compatibility matrix and API to fetch GQL server version

### CODE REVIEW

**Review by SarthakJain26 — CHANGES_REQUESTED** [review]

File: pkg/cmd/config/setAccount.go [hunk 1]
```diff
@@ -158,6 +158,32 @@ var setAccountCmd = &cobra.Command{
...
+
+				if !isCompatible {
+					utils.Red.Println("\n🚫 ChaosCenter version: " + serverResp.Data.GetServerVersion.Value + " is not compatible with the installed LitmusCTL version: " + os.Getenv("CLIVersion"))
+					utils.White_B.Println("Compatible ChaosCenter versions are: ")
+					utils.White_B.Print("[ ")
+					for _, v := range compatibilityArr {
+						utils.White_B.Print("'" + v + "' ")
+					}
+					utils.White_B.Print("]")
+					utils.White_B.Println("")
```
Comment (SarthakJain26) [inline pkg/cmd/config/setAccount.go:181]: Is this required?

File: pkg/cmd/version/version.go [hunk 2]
```diff
@@ -29,6 +29,14 @@ var VersionCmd = &cobra.Command{
...
 	Run: func(cmd *cobra.Command, args []string) {
+		compatibilityArr := utils.CompatibilityMatrix[os.Getenv("CLIVersion")]
 		utils.White_B.Println("Litmusctl version: ", os.Getenv("CLIVersion"))
+		utils.White_B.Println("Compatible ChaosCenter versions: ")
+		utils.White_B.Print("[ ")
+		for _, v := range compatibilityArr {
+			utils.White_B.Print("'" + v + "' ")
+		}
+		utils.White_B.Print("]")
+		utils.White_B.Println("")
```
Comment (SarthakJain26) [inline pkg/cmd/version/version.go:40]: Same

File: pkg/cmd/config/setAccount.go (hunk 1 above)
Comment (SarthakJain26) [reply pkg/cmd/config/setAccount.go:181]: Instead can we have `utils.White_B.Print("]\n")`

**Review by SarthakJain26 — APPROVED** [review]

File: pkg/cmd/config/setAccount.go [hunk 3]
```diff
@@ -158,6 +158,31 @@ var setAccountCmd = &cobra.Command{
...
 
+			serverResp, err := apis.GetServerVersion(authInput.Endpoint)
+			var isCompatible bool
+			if err != nil {
+				utils.Red.Println("\nError: ", err)
+			} else {
+				compatibilityArr := utils.CompatibilityMatrix[os.Getenv("CLIVersion")]
+				for _, v := range compatibilityArr {
+					if v == serverResp.Data.GetServerVersion.Value {
+						isCompatible = true
```
Comment (Jonsy13) [inline pkg/cmd/config/setAccount.go:169]: we can add a `break` also here

**Review by Jonsy13 — APPROVED** [review]

**Review by imrajdas — APPROVED** [review]

**Pull request merged**